install.md

```diff
@@ -54,6 +54,10 @@ It is assumed you are running a Linux machine.
 
 ## Install packaged versions of CRI-O
 
+**The following instructions apply to CRI-O versions  `< 1.29`. Please follow
+the instructions in the [cri-o/packaging repository](https://github.com/cri-o/packaging)
```

Comment:
as we move over there I think we'll want to do some duplication so users don't have to comb through a sea of links, but I am fine with this for now